### DISCUSSION TITLE
how do you cancel restart if auto restart is set on a delay?

### DISCUSSION THREAD

**Discussion by icescoop12** [rfvgyhn/min-ed-launcher#135] [Q&A]
hi added /restart 5 for elite to auto restart ,but how do I disable or cancel the restart if i don't want it to without having to close minimal ed launcher?
thanks

**Comment by rfvgyhn** [comment]
You can press any key besides space. The window does need to be in focus for it to work.

There should be a message that's displayed that says how to cancel it. Is it not showing up for you or maybe it's just restarting too quickly to notice?

I put my restart value at 10 which gives me plenty of time to cancel when I'm done. I then press `space` to restart which gives me an almost immediate restart time.

**Reply by icescoop12** [reply]
in vr should it show?

**Reply by rfvgyhn** [reply]
It shows in the in the window that pops up when you start the game. 

![min-ed-restart](https://github.com/user-attachments/assets/5d78cbc6-3f1e-44f0-9a0f-71885f12390c)

Not sure what your VR setup is like, so maybe you never see it, but you can't cancel the restart without that window being in focus.

**Reply by icescoop12** [reply]
ok i can set voice attacks to focus on that window even shutdown elite and then focus on that window then i can say 'cancel restart' and it will send a command to that window?

**Reply by rfvgyhn** [reply]
I've never used voice attack so I can't say for sure, but that sounds like it would work. It's possible voice attack can inject key press events without needing the window in focus too so you might not have to focus the window first.
